src/main/scala/io/github/davidgregory084/TpolecatPlugin.scala

```diff
@@ -57,7 +57,8 @@ object TpolecatPlugin extends AutoPlugin {
     ScalacOption("-language:experimental.macros", removedIn = Some(V3_0_0)),                              // Allow macro definition (besides implementation and application)
     ScalacOption("-language:higherKinds", removedIn = Some(V3_0_0)),                                      // Allow higher-kinded types
     ScalacOption("-language:implicitConversions", removedIn = Some(V3_0_0)),                              // Allow definition of implicit functions called views
-    ScalacOption("-language:existentials,experimental.macros,higherKinds,implicitConversions", addedIn = Some(V3_0_0)), // the four options above, dotty style
+    ScalacOption("-source future", addedIn = Some(V3_0_0)),                                                       // Emit warnings for features that are planned to be removed (e.g. extending non-open classes outside their files).
+    ScalacOption("-language:existentials,experimental.macros,higherKinds,implicitConversions,strictEquality", addedIn = Some(V3_0_0)), // Require CanEqual for equality checks + the four options above, dotty style
```

Comment:
I'm now wondering if we should split this: if someone wants to opt out of a single option, they need to first remove the whole thing, then add all the other options again. If we split this into separate `-language:foo` flags, it's much easier to remove only the ones you want removed.

Reply:
Also, I just hit https://github.com/lampepfl/dotty/issues/10850 so I no longer like the strict equality option for now.